lib/property_sets/active_record_extension.rb

```diff
@@ -5,6 +5,8 @@
 module PropertySets
   module ActiveRecordExtension
     module ClassMethods
+      attr_accessor :property_sets_connection_class
```

Comment:
I don't remember where we landed on throwing a warning if `property_sets_connection_class=` gets called more than once?

```ruby
      def property_sets_connection_class=(value)
        warn "property_sets_connection_class has already been set!" if @property_sets_connection_class
        @property_sets_connection_class = value
      end
      ```

Reply:
It seems a little too complicated to set up and make work with Rails 6.0, tbh.

Reply:
Ah, right. Now I remember there was some issue specific to 6.0. I don't remember what it was through 😅. But, I agree that this small nicety isn't worth the additional effort 😄